torch/_dynamo/variables/functions.py

```diff
@@ -1590,12 +1599,56 @@ def python_type(self) -> type:
         return types.FunctionType
 
     def get_function(self) -> types.FunctionType:
+        from .base import AsPythonConstantNotImplementedError, ClosureConversionError
+
+        if self._converting:
```

Comment:
nit: could we just use a set (of id()) instead of adding a new field?

Reply:
Done